File: CPP/PetShop.cpp
#include <iostream>
#include <string>

using namespace std;

class PetShop{
    private:
        string id;
        string nama_produk;
        int harga_produk;
        int stok_produk;
    public:
        PetShop(string id, string name, int harga, int stok){
            this->id = id;
            this->nama_produk = name;
            this->harga_produk = harga;
            this->stok_produk = stok;
        }

        // Method-method setter dan getter
        void setId(string id){
            this->id = id;
        }
        string getId(){
            return this->id;
        }

        void setNama(string nama){
            this->nama_produk = nama;
        }
        string getNama(){
            return this->nama_produk;
        }

        void setHarga(int harga){
            this->harga_produk = harga;
        }
        int getHarga(){
            return this->harga_produk;
        }

        void setStok(int stok){
            this->stok_produk = stok;
        }
        int getStok(){
            return this->stok_produk;
        }


        ~PetShop(){}
};

File: PHP/Program/PetShop.php
<?php
class PetShop{
    private $id;
    private $nama_produk;
    private $harga_produk;
    private $stok_produk;

    public function __construct($id, $nama, $harga, $stok) {
        $this->id = $id;
        $this->nama_produk = $nama;
        $this->harga_produk = $harga;
        $this->stok_produk = $stok;
    }

    public function setId($id){
        $this->id = $id;
    }
    public function getId(){
        return $this->id;
    }

    public function setNama($nama){
        $this->nama_produk = $nama;
    }
    public function getNama(){
        return $this->nama_produk;
    }

    public function setHarga($harga){
        $this->harga_produk = $harga;
    }
    public function getHarga(){
        return $this->harga_produk;
    }

    public function setStok($stok){
        $this->stok_produk = $stok;
    }
    public function getStok(){
        return $this->stok_produk;
    }
}